app/templates/events/view/sessions.hbs

```diff
@@ -0,0 +1,103 @@
+<div class="ui basic buttons">
+  <button class="ui button">All</button>
+  <button class="ui button">Pending</button>
+  <button class="ui button">Accepted</button>
+  <button class="ui button">Confirmed</button>
+  <button class="ui button">Rejected</button>
+</div>
+<button class="ui teal button">Create Session</button>
+<select class="ui dropdown">
```

Comment:
Not this dropdown @harshitagupta30 . A dropdown button. Like how you did for the accept and reject